### PR TITLE
Use ubuntu 1804 arm64 queue outerloop

### PR DESCRIPTION
**Pull request by jashook**

### CODE REVIEW

File: eng/platform-matrix-managed-test-build.yml [hunk 1]
```diff
@@ -73,8 +73,12 @@ jobs:
         image: ubuntu-16.04-cross-arm64-cfdd435-20190923200213
         registry: mcr
       helixQueues:
+      # Standardize on one set of hardware, when we can handle load in one queue.
+      # - ${{ if and(eq(variables['System.TeamProject'], 'public'), eq(parameters.helixQueueGroup, 'pr')) }}:
+      #   - (Ubuntu.1804.Arm64.Open)Ubuntu.1604.Arm64.Docker.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm64v8-a45aeeb-20190620155855
+      # - ${{ if and(eq(variables['System.TeamProject'], 'public'), ne(parameters.helixQueueGroup, 'pr')) }}:
       - ${{ if eq(variables['System.TeamProject'], 'public') }}:
-        - (Ubuntu.1804.Arm64.Open)Ubuntu.1604.Arm64.Docker.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm64v8-a45aeeb-20190620155855
+        - (Ubuntu.1604.Armarch.Open)Ubuntu.1604.Armarch.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm64v8-a45aeeb-20190620155855
```
Comment (jkoritzinsky) [inline eng/platform-matrix-managed-test-build.yml]: This should still be 1804 since the docker image is an 18.04 image.

**Comment by jashook** [comment]
thanks!

**Comment by jashook** [comment]
Spoke with @echesakovMSFT offline and addressed his suggestions. Merging to save rerunning ci on a non-functional change

**Comment by jashook** [comment]
Looks like our armarch queue is not large enough to handle the volume of arm64 and arm32 testing.

**Pull request merged**